{{cookiecutter.repo_name}}/Makefile

```diff
@@ -27,7 +32,14 @@ help:
 clean:
 	find . -name '*.pyc' -delete
 	coverage erase
-	rm -rf assets
+	rm -rf {{cookiecutter.repo_name}}/assets {{cookiecutter.repo_name}}/static/build coverage htmlcov
+
+requirements.js:
+	npm install
+	$(NODE_BIN)/bower install
+
+requirements: requirements.js
+	pip install -qr requirements/local.txt --exists-action w
```

Comment:
You don't need to add a new `requirements` target, you just need to add a new dependency in the existing targets below.